hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileAppendHandler.java

```diff
@@ -122,4 +132,25 @@ public void handle(@NonNull final HandleContext context) throws HandleException
         It will not be committed to state until commit is called on the state.--- */
         fileStore.put(fileBuilder.build());
     }
+
+    private static void handleAppendUpgradeFile(FileAppendTransactionBody fileAppend, HandleContext handleContext) {
```

Comment:
Minor note, making every private method static makes unit tests harder, particularly if you need to spy the class to force an exception.

Reply:
by mistake sorry I thought it was used also out of the class. fixed it